vue-components/stories/Message.stories.ts

```diff
@@ -53,6 +37,21 @@ controllableMessage.argTypes = {
 	content: {
 		control: {
 			type: 'text',
-		}
+		},
 	},
 };
+
+export function allMessageTypes(): Component {
```

Comment:
Same comment as above, only here according to the structure guideline, the story should be called `All`.

Reply:
I interpreted the outcome of this [discussion](https://mattermost.wikimedia.de/swe/pl/i5ik69ijzpdt9jf3cr6z98ocyh) as it being agreed that it makes sense for the story titles in the menu to be more specific if possible. But maybe I misread it?

Reply:
Oh, I didn't realize it was concluded, If @SaiSan-WMDE agrees, then I approve as well :)

Reply:
After giving it some more thought: being specific (but yet consistent) would mean using either "All" (contains: flavors, disabled, validation states...in case of buttons, for example) or "All types" (component typologies) as titles for these sections, and I'm not sure how much helpful adding the word "type" in certain cases would really be? So I'd say that we should start using only the word "All" in every section. Do you agree?

Reply:
If the actual desire is to always have all permutations in a single page, then I will implement that. Will update the PR.

Reply:
Done

Reply:
thanks, Michael! 🙏